Assets/PyroParticles/Prefab/Script/MeteorSwarmScript.cs

```diff
@@ -121,6 +133,13 @@ private IEnumerator SpawnMeteor()
                 }
             }
         }
+        
+        //Destroy the meteor target indicator
+        private IEnumerator DestroyTargetIndi(GameObject obj)
+        {
+            yield return new WaitForSeconds(targetDelay);
```

Comment:
I don't know how you got your code to run, lol. I get a compiler error because you named the variable `delay` instead of `targetDelay`. Be sure to fix that :)

Reply:
Lol whoops. I forgot to run this right after I made a minor change in the script....